rpc-server/src/dispatchers/blockchain.rs

```diff
@@ -429,6 +429,8 @@ impl BlockchainInterface for BlockchainDispatcher {
     async fn get_transactions_by_address(
         &mut self,
         address: Address,
+        from: Vec<Address>,
+        to: Vec<Address>,
         max: Option<u16>,
     ) -> RPCResult<Vec<ExecutedTransaction>, (), Self::Error> {
```

Comment:
Instead of adding it in the middle of the existing arguments in the existing method, which is a breaking and backward-incompatible change, I'd instead create a new method that explicitly searches for transactions with filters. Maybe it can be possible to pass a `filters` object as an argument, that is freely extensible, instead of fixed arguments?

Reply:
We tried that, but we ran into problems serialising / deserialising with clap. I would really like to make this work, but I could not find a solution. I asked @jsdanielh and @ii-cruz, but they came to the same conclusion. 

Another solution was to try to make our own parser, but I don't know how to do that.

And yes, I agree with using an object, so in the future we could add other filters like `block_ranges`.

Reply:
Then at least I'd make this a new method `get_transactions_by_address_with_filters` and add the new arguments there, as I think they will be an edge-case usage.

Reply:
Ok. I thought it would be more common use case.